### PR TITLE
Fix avatars not overlapping

### LINKED ISSUE #6399: [CAL-750] avatars not showing (+N) people
I have a team link with 4 members:

but only 3 avatars (1 logo + 2 avatars) are being shown:

they used to overlap and there was also a (+2) Icon if I believe

<sub>From [SyncLinear.com](https://synclinear.com) | [CAL-750](https://linear.app/calcom/issue/CAL-750/avatars-not-overlapping-and-not-showing-n-people)</sub>

### PR DESCRIPTION
**Pull request by sean-brydon**
## What does this PR do?

Fixes avatars not overlapping and using the old version.

Fixes #6399 

CleanShot 2023-01-11 at 10 46 11@2x
CleanShot 2023-01-11 at 10 47 10@2x

## Type of change

- [X] Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

File: apps/web/components/booking/UserAvatars.tsx [hunk 1]
```diff
@@ -38,7 +36,7 @@ export const UserAvatars = ({
           href?: string;
         }[]
       }
-      size={props.size}
+      size="sm"
```
Comment (sean-brydon) [inline apps/web/components/booking/UserAvatars.tsx:39]: We only use this as small

File: apps/web/pages/event-types/index.tsx [hunk 2]
```diff
@@ -512,12 +511,12 @@ const EventTypeListHeading = ({
   teamId,
 }: EventTypeListHeadingProps): JSX.Element => {
   return (
-    <div className="mb-4 flex">
+    <div className="mb-4 flex items-center space-x-2">
```
Comment (sean-brydon) [inline apps/web/pages/event-types/index.tsx:514]: Fix the spacing from avatar -> eventtype list heading

File: packages/ui/components/avatar/AvatarGroup.tsx [hunk 3]
```diff
@@ -11,19 +11,21 @@ export type AvatarGroupProps = {
   }[];
   className?: string;
   accepted?: boolean;
+  truncateAfter?: number;
 };
 
 export const AvatarGroup = function AvatarGroup(props: AvatarGroupProps) {
   const avatars = props.items.slice(0, 4);
   const LENGTH = props.items.length;
+  const truncateAfter = props.truncateAfter || 4;
```
Comment (sean-brydon) [inline packages/ui/components/avatar/AvatarGroup.tsx:20]: Make this more reusable - we often use 4 but makes this available if we ever need more or less

File: packages/ui/components/avatar/index.ts [hunk 4]
```diff
@@ -1,3 +1,4 @@
 export { Avatar } from "./Avatar";
 export type { AvatarProps } from "./Avatar";
 export { AvatarGroup } from "./AvatarGroup";
+export type { AvatarGroupProps } from "./AvatarGroup";
```
Comment (sean-brydon) [inline packages/ui/components/avatar/index.ts:4]: Fixing props not being exported by calcomui

File: packages/ui/components/avatar/AvatarGroup.tsx [hunk 5]
```diff
@@ -34,14 +36,14 @@ export const AvatarGroup = function AvatarGroup(props: AvatarGroupProps) {
...
-                  <span>+{LENGTH - 3}</span>
+                  <span>+{LENGTH - truncateAfter - 1}</span>
                 </div>
               </li>
             );
           }
           // Always display the first Four items items
           return (
-            <li key={enumerator} className="-mr-4 inline-block">
+            <li key={enumerator} className="-mr-3 inline-block">
```
Comment (sean-brydon) [inline packages/ui/components/avatar/AvatarGroup.tsx:46]: -mr-4 felt like too much - couldnt see avatars that well

**Comment by Jaibles** [comment]
@sean-brydon can you just double-check how much the overlap is? Should be `4px`
CleanShot 2023-01-11 at 12 44 38@2x

Kinda looks like in the screenshot the person in the middle is getting really sandwiched :)
CleanShot 2023-01-11 at 12 45 23@2x

**Review by emrysal — APPROVED** [review]
LGTM 🎸

**Pull request merged**